frontend/src/features/sketches/SketchModal.tsx

```diff
@@ -0,0 +1,10 @@
+import { Modal } from "~/shared/components/Modal";
+type Props = {
+    selectedSketchUrl: string;
+    setSelectedSketchUrl: (url: string | null) => void;
```

Comment:
[nits]
インデント揃えておこう！
```suggestion
  selectedSketchUrl: string;
  setSelectedSketchUrl: (url: string | null) => void;
```